python/orca/src/bigdl/orca/learn/mpi/mpi_runner.py

```diff
@@ -59,8 +59,10 @@ def get_ip(iter):
         print("Hosts: ", self.hosts)
         self.processes_per_node = processes_per_node
         self.env = env if env else {}
+        import getpass
```

Comment:
put this into the beginning with other imports?

Reply:
ok